client/themes/bcgov/assets/i18n/en.json

```diff
@@ -15,7 +15,7 @@
     "registration.registries.ca": "Registration",
     "relationship.registries.ca": "Relationship",
     "address.registries.ca": "Address",
-    "demo.business_number.registries.ca": "Business Number",
+    "business_number.registries.ca": "Business Number",
```

Comment:
Did you mean to include this change?

Reply:
Yes, it needs to go in for the new BN credential, and can go in ahead of time with no impact.  (The changes for the BN credential are all in the BC Reg repository)